autoreject/tests/test_autoreject.py

```diff
@@ -318,3 +319,33 @@ def test_io():
     assert_array_equal(epochs_clean1.get_data(), epochs_clean3.get_data())
     assert_array_equal(reject_log1.labels, reject_log2.labels)
     assert_array_equal(reject_log1.labels, reject_log3.labels)
+
+
+def test_fnirs():
+    # Test that autoreject runs on fNIRS data.
+    raw = mne.io.read_raw_nirx(os.path.join(
```

Comment:
Is this too much? We can remove if you think its over the top.

Reply:
how big is the dataset?

Reply:
A total size of 74 MB

Reply:
okay that's fine I think